backend/createStaffAndRooms.py

```diff
@@ -45,81 +46,101 @@ def search_ticket(ticket, guest_entries):
     return False
 
 
-def real_date(a_date):
+def real_date(a_date: str, year=None):
+    """Convert string date into python date
+
+    Args:
+        a_date (str): date string, 
+            expected format: "Mon - 11/7"
+        year (Optional[int]): year, in 4-digit format
+            Allows specification of year, otherwise is in current year at runtime
+    Returns:
+        date: python `date` object
+    """
+    year = year or datetime.now().year
     day, date = a_date.split('-')
     month, day = date.lstrip().split('/')
-    return parse_date("%s-%s-%s" % (datetime.now().year, month, day))
+    return parse_date("%s-%s-%s" % (year, month, day))
 
 def create_rooms_main(rooms_file, is_hardrock=False):
     rooms=[]
     _rooms_fields, rooms_rows = ingest_csv(rooms_file)
-
+    # validate imported rooms list
+    
+    # rooms_import_list = [RoomPlacementListIngest(**r) for r in rooms_rows]
+    validation_errors = []
+    rooms_import_list = []
+    for r in rooms_rows:
+        try:
+            room_data = RoomPlacementListIngest(**r)
+            rooms_import_list.append(room_data)
+        except ValidationError as e:
+            print(f"Validation error for row {r}: {e}")
```

Comment:
I'm :ok: with the non-fatal validation errors but only because I'm making changes to support in-place updates of rooms - as opposed to the original intent of rooms being only set a single time.

Reply:
This should be a debug log...